### ISSUE TITLE
drive_mkdir, overwrite = FALSE not working as expected on a shared drive

### ISSUE THREAD

**Issue by griswomw** [tidyverse/googledrive#324]
I am using drive_mkdir to create new folders on an existing google drive.  When I use the overwrite = FALSE, it will still create multiple copies of a new folder whereas I was expecting it to fail if a folder with that name already existed.

Please include a minimal reproducible example (AKA a reprex). If you've never heard of a [reprex](https://reprex.tidyverse.org/) before, start by reading <https://www.tidyverse.org/help/#reprex>.

---

## I would expect the second time running the same line to fail rather than to create a duplicate folder with the same name

```r
# Run First Time

drive_mkdir("Test", "https://drive.google.com/drive/u/0/folders/SOME_TEST_FOLDER/", overwrite = FALSE)

# Run Same Line Second Time

drive_mkdir("Test", "https://drive.google.com/drive/u/0/folders/SOME_TEST_FOLDER/", overwrite = FALSE)

```

**Comment by griswomw** [comment]
Updating as follows:  overwrite = FALSE seems to work properly when identifying a parent folder by ID on 'My Drive' using path=as_id("some id").  However, the same syntax fails when identifying a parent folder by ID on 'Shared Drives' using path=as_id("some id").  In the latter case, duplicates of the same subfolder name are created.

**Comment by jennybc** [comment]
Probably related to #254 and, fingers crossed, will self-resolve once I do the necessary updates

**Comment by jennybc** [comment]
OK, no, not related to #254. I can replicate this, but the fix will be separate, i.e. after I address #254.

**Comment by jennybc** [comment]
Fixed by 021102337a2faf5f9d758fccea76d4e6573f60a7, not by 76d070594fc37fa31752ef95c14450fe32a0a374

**Comment by griswomw** [comment]
Thank you so much!!

On Sat, Apr 10, 2021 at 4:52 PM Jennifer (Jenny) Bryan <
***@***.***> wrote:

> Fixed by 0211023
> <https://clicktime.symantec.com/3USqzdC85P5oNmRC9QzfGsm7Vc?u=https%3A%2F%2Fgithub.com%2Ftidyverse%2Fgoogledrive%2Fcommit%2F021102337a2faf5f9d758fccea76d4e6573f60a7>,
> not by 76d0705
> <https://clicktime.symantec.com/35JgrzgimJiRNNfU7gNm4fi7Vc?u=https%3A%2F%2Fgithub.com%2Ftidyverse%2Fgoogledrive%2Fcommit%2F76d070594fc37fa31752ef95c14450fe32a0a374>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://clicktime.symantec.com/3Au8sxVUX8GoUR3fwhH5an47Vc?u=https%3A%2F%2Fgithub.com%2Ftidyverse%2Fgoogledrive%2Fissues%2F324%23issuecomment-817207120>,
> or unsubscribe
> <https://clicktime.symantec.com/3UitWwaTHbQskF8PdQZhamm7Vc?u=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FANYCZIJPQFCUU4TMXXYCLJTTIDCATANCNFSM4XBJQ35A>
> .
>
-- 

*Michael Griswold*
*Managing Director of Strategy and Risk Management*

Ascension Investment Management
4600  Edmundson Rd. | St. Louis, MO 63134
Phone: (314) 733-8318
***@***.***


CONFIDENTIALITY NOTICE: This email message and any accompanying data or files is confidential and may contain privileged information intended only for the named recipient(s). If you are not the intended recipient(s), you are hereby notified that the dissemination, distribution, and or copying of this message is strictly prohibited. If you receive this message in error, or are not the named recipient(s), please notify the sender at the email address above, delete this email from your computer, and destroy any copies in any form immediately. Receipt by anyone other than the named recipient(s) is not a waiver of any attorney-client, work product, or other applicable privilege.
